ui/utils/hooks/useMeshModelComponents.js

```diff
@@ -1,5 +1,7 @@
 import _ from "lodash";
 
+export const WILDCARD_V = "*(All Versions)"
```

Comment:
Would it be ok if we displayed this string?

```suggestion
export const WILDCARD_V = "All Versions"
```